### PR TITLE
dev-Airat

### PR DESCRIPTION
**Pull request by Airat163**
check please my code, i have two commits

### CODE REVIEW

File: src/main/java/com/example/demo/TestGitFork.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.example.demo;
+
+public class TestGitFork {
+    public void forking(String message) {
```
Comment (taron1991) [inline src/main/java/com/example/demo/TestGitFork.java:4]: here

File: src/main/java/com/example/demo/TestGitFork.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (taron1991) [inline src/main/java/com/example/demo/TestGitFork.java]: add here sout with concatenation

**Comment by taron1991** [comment]
add sout in method

File: src/main/java/com/example/demo/TestGitFork.java (hunk 1 above)
Comment (Airat163) [reply src/main/java/com/example/demo/TestGitFork.java:4]: i have just changed my code

Comment (taron1991) [reply src/main/java/com/example/demo/TestGitFork.java]: Airat add a loop 10 and use array of strings

**Comment by taron1991** [comment]
Airat add a loop 10 and use array of strings

File: src/main/java/com/example/demo/TestGitFork.java (hunk 1 above)
Comment (taron1991) [reply src/main/java/com/example/demo/TestGitFork.java]: ШФКЕФ

**Comment by Airat163** [comment]
i add loop

**Comment by taron1991** [comment]
change void to array and rename your method to loopArray

**Comment by Airat163** [comment]
i finished

File: src/main/java/com/example/demo/TestGitFork.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.example.demo;
+
+public class TestGitFork {
+    public String[] loopArray(String message) {
+        String[] strings = {"Airat", "Aigel", "Taron", "Vasya"};
```
Comment (taron1991) [inline src/main/java/com/example/demo/TestGitFork.java:5]: change Vasya to Zahar

**Review by taron1991 — COMMENTED** [review]
change Vasya to Zahar

File: src/main/java/com/example/demo/TestGitFork.java (hunk 3 above)
Comment (Airat163) [reply src/main/java/com/example/demo/TestGitFork.java]: i changed

**Comment by taron1991** [comment]
create new class Batch, create 2 methods,methodX and methodY

**Comment by Airat163** [comment]
i create class

**Pull request merged**